code/modules/atmospherics/atmospherics.dm

```diff
@@ -27,9 +27,8 @@ Pipelines + Other Objects -> Pipe network
 	var/initialize_directions = 0
 	var/pipe_color
 
-	var/global/datum/pipe_icon_manager/icon_manager
-	var/obj/machinery/atmospherics/node1
-	var/obj/machinery/atmospherics/node2
+	var/list/nodes = list() // list of adjacent atmos devices to which we are connected
```

Comment:
As discussed, might be good to merge these lists and LAZYLIST them (or assign them in Initialize()).

Reply:
Done.